### DISCUSSION TITLE
Is it possible to precompile the modules?

### DISCUSSION THREAD

**Discussion by Blank-c** [pyinstaller/pyinstaller#6976] [Help]
Some of the pyd files are flagged as malware in virustotal so I want to precompile the pyd files and run some experiments on them and later include them in my code. Possible? If yes, how?

**Comment by bwoodsend** [comment]
A pyd is a compiled extension module so it's already compiled. PyInstaller doesn't do anything to them except copy them into your build.

**Reply by Blank-c** [reply]
oh
